logic.js

```diff
@@ -32,12 +32,12 @@ app.post("/scores/add", function (req, res) {
 })
 
 app.post("/player/add", function (req, res){
-  if (req.query.player === undefined) {
+  if ((req.query.player === "") || (req.query.player === undefined)) {
```

Comment:
Vill du använda lite cool JS-magi kan du ändra detta till:
```suggestion
  if (!req.query.player) {
```
På så sätt fångar du även upp om det skulle vara `null`. Om du är mer nyfiken på varför/hur det fungerar så kan du läsa lite om automatic type conversion i js (https://www.sitepoint.com/automatic-type-conversion/)